content/influxdb/v2.0/write-data/developer-tools/csv.md

```diff
@@ -64,6 +64,23 @@ To test the CSV to line protocol conversion process, use the `influx write dryru
 command to print the resulting line protocol to stdout rather than write to InfluxDB.
 {{% /note %}}
 
+{{% note %}}
+##### "too many open files" errors
+
+When attempting to write large amounts of CSV data into InfluxDB, you might see an error like the following:
+
+```
+Error: Failed to write data: unexpected error writing points to database: [shard 36] fcntl: too many open files.
+```
+
+Fix this error by increasing the number of open files allowed by the OS.
+The following command works on both Linux and MacOS:
```

Comment:
To fix this error, run the following command to increase....(works on both Lin....):